src/main/npl-1.0/processes/settle.npl

```diff
@@ -6,7 +6,7 @@ use objects.iou.Iou
 @api
 protocol[iouOwner, carOwner] Settle(var iou: Iou, var car: Car) {
     @api
-    permission[iouOwner | carOwner] swap() {
+    permission[iouOwner & carOwner] swap() {
```

Comment:
the npl api doesn't support passing two parties to a permission. It can from the language point of view those are the three options
- iouOwner | carOwner
- iouOwner
- carOwner

Reply:
Fixed this, thanks for the feedback